config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml

```diff
@@ -391,10 +391,10 @@ presubmits:
           privileged: true
         resources:
           limits:
-            cpu: "4"
+            cpu: "7"
             memory: 9000Mi
```

Comment:
TODO: check how much memory building k/k for e2e uses these days, this matches the k/k presubmits but we probably should grant more memory given the amount of CPU given (~1GB/core is super low ...) and memory bloat over time ...

Reply:
One tip would be to migrate them to `eks-prow-build-cluster` so we can leverage https://monitoring-eks.prow.k8s.io/ to monitor resources consumption.

Reply:
+1 as followup?

Reply:
I want it in the same cluster as the k/k presubmit because we need to catch any problems versus the envrionment we actually run k/k presubmits with first and foremost.